lib/run/summary.js

```diff
@@ -2,7 +2,11 @@ var _ = require('lodash'),
     SerialiseError = require('serialised-error'),
 
     proxyPropertyList,
-    RunSummary;
+    RunSummary,
+    exportReference = {
```

Comment:
Yes. We are changing app to export to this format. And at a later date we will address import of globals in older format the right way once we have streamlined the options module and started using variable list.